src/Logic.hs

```diff
@@ -369,16 +369,29 @@ handlePullRequestEdited prId newTitle newBaseBranch state =
     -- Do nothing if the pull request is not present.
     Nothing -> pure state
 
+data ParseResult a
+  = Success a
+  | Unknown Text
+  | Ignored
```

Comment:
Nice datatype, can you comment it a bit more, as well as the `isSuccess` function.